packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -63,6 +63,7 @@ const ALTER_RE = /^(\s*alter\s)/i
 declare global {
   // eslint-disable-next-line no-var
   var NODE_CLIENT: true
+  const TARGET_ENGINE_TYPE: 'binary' | 'library' | 'data-proxy' | 'all'
```

Comment:
Since these would leak as global vars (right?), maybe it would be better to prefix them with `PRISMA_`

Reply:
I think these are always stripped by the compiler when value is provided, that is the case for `NODE_CLIENT` for sure.

Reply:
So I guess it just matters in the tests?

Reply:
Yes, that's correct, those constant never actually exist in compiled bundles.